.changeset/famous-panthers-occur.md

```diff
@@ -0,0 +1,7 @@
+---
+'@sap-ux/adp-tooling': minor
+'@sap-ux/preview-middleware': patch
```

Comment:
Please remove preview-middleware and axios-extension from the changeset.

Reply:
Done